src/content/glossary/index.md

```diff
@@ -27,12 +27,11 @@ An object containing an [address](#address), balance, [nonce](#nonce), and optio
 
 ### address {#address}
 
-Most generally, this represents an [EOA](#eoa) or [contract](#contract-account) that can receive (destination address) or send (source address) [transactions](#transaction) on the blockchain. More specifically, it is the rightmost 160 bits of a [Keccak hash](#keccak-256) of an [ECDSA](#ecdsa) [public key](#public-key).
+A string of 40 characters representing an [EOA](#eoa) or [contract](#contract-account) account in Ethereum's state tree. More specifically, it is the rightmost 160 bits of a [Keccak hash](#keccak-256) of an [ECDSA](#ecdsa) [public key](#public-key).
```

Comment:
```suggestion
A string of characters starting with `0x` representing an [EOA](#eoa) or [contract](#contract-account) account in Ethereum's state tree. 
```

I think this could be a more simplified version. Let me know what you think @konopkja

Reply:
```suggestion
A unique identifier that represents your account. All accounts on Ethereum have a public address that starts with `0x`. This address is sharable and used to receive assets to your account. 
```

@konopkja put together in call